README.md

```diff
@@ -54,10 +54,10 @@ library implemented in Java, with ports to other languages.
 
 | Module                                                                                    | Description
 | ----------------------------------------------------------------------------------------- | -----------
-| [SCodes](https://github.com/scytheStudio/SCodes)                                          | Qt & Qml wrapper
+| [SCodes](https://github.com/scytheStudio/SCodes)                                          | Qt & Qml wrapper, powered by [nu-book/zxing-cpp](https://github.com/nu-book/zxing-cpp)
```

Comment:
Why do we need this here? no big deal but seems like a change to suggest for SCodes

Reply:
Sure, I'll remove it. I just noticed this new entry, was curious what it was based on and noticed this fact. Similar to other entries below, I thought this "powered by" info might be helpful (and yes, I'm biased ;)).